config/build/rollup.config.js

```diff
@@ -170,8 +174,15 @@ const createConfig = (
                 browser: platform === "browser",
                 extensions,
             }),
-            autoExternal({
+            nodeExternals({
```

Comment:
Switched to this plugin as this is was wonder-blocks uses, its actively maintained (autoExternals hadn't been touched in 5 years), and it allows us to selectively force bundling.

Reply:
Reverted this now that we're just making this a regular old package.